tests/search/components/TestSearchForm.js

```diff
@@ -4,27 +4,32 @@ import ReactTestUtils from 'react-addons-test-utils';
 import SearchForm from 'search/components/SearchForm';
 
 const render = ReactTestUtils.renderIntoDocument;
-const findByTag = ReactTestUtils.findRenderedDOMComponentWithTag;
 
 describe('<SearchForm />', () => {
+  let onSearch;
+  let root;
+  let form;
+  let input;
+
+  beforeEach(() => {
+    onSearch = sinon.spy();
+    root = render(<SearchForm onSearch={onSearch} />);
+    form = root.refs.form;
+    input = root.refs.query;
+  });
+
   it('renders a form', () => {
-    const root = render(<SearchForm onSearch={sinon.spy()} />);
-    const form = findByTag(root, 'form');
     assert.ok(form.classList.contains('search-form'));
   });
 
   it('renders a search input', () => {
-    const root = render(<SearchForm onSearch={sinon.spy()} />);
-    const input = findByTag(root, 'input');
     assert.equal(input.placeholder, 'Search');
-    assert.equal(input.type, 'text');
+    assert.equal(input.type, 'search');
   });
 
   it('calls onSearch with a search query', () => {
-    const onSearch = sinon.spy();
-    const root = render(<SearchForm onSearch={onSearch} />);
     root.refs.query.value = 'adblock';
```

Comment:
`beforeEach` has `input = root.refs.query` so this could be `input.value` instead.